constraint/pkg/client/drivers/local/args.go

```diff
@@ -98,3 +99,19 @@ func DisableBuiltins(builtins ...string) Arg {
 		d.capabilities.Builtins = nb
 	}
 }
+
+<<<<<<< HEAD
```

Comment:
Merge conflict indicators

Reply:
Yep, I tried to rebase off [Will's refactor](https://github.com/willbeason/frameworks/commit/7078cd84b1f2f4f99a5b7acc79e000f88d26f214) which fully moves `Handler` into it's own package, but quickly regretted it. Without Will's client test changes, there will already exist a `handler` type in client. We would have to make a bunch of named imports, only to clean them up later once Will's client changes are merged in.

I think the easiest solution for now is to copy the `TargetHandler` file into the new `handler` package, and defer the proper refactor until Will's PR

@willbeason does this sound reasonable?